.github/workflows/lint.yml

```diff
@@ -1,13 +1,14 @@
 name: Lint
 
 on:
-  # Trigger the workflow on push or pull request,
-  # but only for the main branch
   push:
     branches: [main]
   pull_request:
-    branches:
-      - "*"
+    paths-ignore:
```

Comment:
For linter, do we still ignore these paths? I thought it might still catch white space issues for example.